applications/search/src/app/breadcrumbs/informationModel-breadbrumb/information-breadcrumb.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { resolve } from 'react-resolver';
+import { getInformationModel } from '../../../api/get-information-model';
+import { getTranslateText } from '../../../lib/translateText';
+
+const memoizedGetInformationModel = _.memoize(getInformationModel);
```

Comment:
memoize lagrer deg en ukontrollerbar funksjonell cache som skal holde alle dine request-response.

hvis det er viktig å demo akkurat nå, du kan jo merge det in, men da må du endre det til redux senere.

Jeg anbefaler å intrudusere redux (standard copypaste fra noe?, eller skal vi konvertere det til lib nå?) før breadcrumb. da trenger du ikke å endre breadcrumb etterpå

Reply:
Jeg tror vi kan se på det senere, gjøre det likt for alle detaljsider.